src/main/java/edu/greenblitz/robotName/subsystems/Dashboard.java

```diff
@@ -0,0 +1,67 @@
+package edu.greenblitz.robotName.subsystems;
+
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import edu.greenblitz.robotName.subsystems.prototypes.Prototypes;
+import edu.greenblitz.robotName.utils.GBSubsystem;
+import edu.greenblitz.robotName.utils.tuneableNumber.AutoSetterTunableNumber;
+import edu.greenblitz.robotName.utils.tuneableNumber.TunableNumber;
+import edu.greenblitz.robotName.utils.tuneableNumber.TunableNumberManager;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+import static edu.greenblitz.robotName.subsystems.Dashboard.motorType.SPARK_MAX;
+import static edu.greenblitz.robotName.subsystems.Dashboard.motorType.TALON_SRX;
+
+public class Dashboard extends GBSubsystem {
+	private static Dashboard instance;
+	public static Dashboard getInstance() {
+		init();
+		return instance;
+	}
+	
+	public static void init() {
+		if (instance == null) {
+			instance = new Dashboard();
+		}
+	}
+	
+	private Dashboard() {
+		prototypesDashboard();
+	}
+	
+	
+	public static String prototypesTabName = "Prototypes";
```

Comment:
consider changing the variable name from prototypesTabName to prototypesTabTitle because it's the title of the tab
```suggestion
	public static String prototypesTabTitle = "Prototypes";
```